recipes/deltalake/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "deltalake" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deltalake-{{ version }}.tar.gz
+  sha256: b8793a1c7a1219c8935ed9926a6870c9341597396c0f50bf6ff98c323d06ee0f
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - maturin >=0.14,<0.15
+    - make
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [linux]
```

Comment:
Noting that this is included (along with `xz` in the host dependencies) to avoid overlinking warnings during the linux-64 post-build